src/components/Product/Product.tsx

```diff
@@ -0,0 +1,11 @@
+import { ProductSchema } from "@/schemas/ProductSchema"
+
+interface ProductProps {
+  product: ProductSchema
+}
+
+export const Product = ({product}: ProductProps) => {
+  return (
+    <li>{product.title}</li>
+  )
+}
```

Comment:
We usually wouldn't create a component for something as simple as this (you could just use the `li` outside), but I'm guessing you did it because it will be extended later, and it will make more sense for it to exist.

Reply:
Yes, exactly!